examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -79,7 +101,7 @@ function App() {
             ) : (
               <Route path="/">
                 <EmailPassword.EmailPasswordAuth>
-                  <Home />
+                  {isUserOnValidSubdomain ? <Home /> : <Spinner />}
```

Comment:
Time to first render is a pretty crucial metric in SPAs. Adding a backend call before we even render anything simply to check if the user is on the correct subdomain seems like a bad trade-off to me. 

We have such a pattern already for email verification. We decided that it was better to:
 - render the application assuming that email is verified. 
 - verify if that's the case in the useEffect hook.
 - Do nothing if true. Redirect if false.